01-path-basics/101-start-here/templates/lab-ide-novpc.template

```diff
@@ -12,6 +12,18 @@
   "Mappings": {},
   "Conditions": {},
   "Resources": {
+    "EksVpc": {
```

Comment:
So the user will automatically have the EKS VPC created when they create the Lab ENV… none of this cost money correct?